package-scripts.js

```diff
@@ -307,17 +307,17 @@ module.exports = {
         description: 'Build API docs'
       }
     },
-    updateContributors: {
-      script: 'contributors',
-      description: 'Update list of contributors in package.json'
+    updateAuthors: {
+      script: 'scripts/update-authors.js',
+      description: 'Update list of AUTHORS'
     },
     linkifyChangelog: {
       script: 'node scripts/linkify-changelog.js',
       description: 'Add/update GitHub links in CHANGELOG.md'
     },
     version: {
       script:
-        'nps updateContributors && nps linkifyChangelog && git add -A ./package.json ./CHANGELOG.md',
+        'nps updateAuthors && nps linkifyChangelog && git add -A ./AUTHORS ./CHANGELOG.md',
```

Comment:
I think we still need the package.json, it contains the new version number.

Reply:
I didn't publish mocha myself. We update version on `package.json` and then run `npm start version`?

Reply:
[release process](https://github.com/mochajs/mocha/blob/master/MAINTAINERS.md#mochas-release-process)
I don't think so, when I read item nr.4. We run `npm version` and this includes contributors update. Did you test it?

Reply:
Sure, I test them. But I just realize `npm version`. Without `package.json` here, `npm` add `package.json` and commit them. It works.

Reply:
> I think we still need the package.json, it contains the new version number.

Reply:
> I didn't publish mocha myself. We update version on `package.json` and then run `npm start version`?

Reply:
> I think we still need the package.json, it contains the new version number.

Reply:
> I didn't publish mocha myself. We update version on `package.json` and then run `npm start version`?

Reply:
> Sure, I test them. But I just realize `npm version`. Without `package.json` here, `npm` add `package.json` and commit them. It works.

Reply:
> Sure, I test them. But I just realize `npm version`. Without `package.json` here, `npm` add `package.json` and commit them. It works.